src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java

```diff
@@ -39,10 +46,10 @@ public boolean validateUserIdWithException(String userId) {
         return true;
     }
 
-    public boolean validateUserIsPresentWithException(Optional<UserProfile> userProfile, String userId) {
+    public boolean validateUserIsPresentWithException(Optional<UserProfile> userProfile) {
         if (!userProfile.isPresent()) {
             auditService.persistAudit(HttpStatus.NOT_FOUND, ResponseSource.SYNC);
-            final String exceptionMsg = String.format("%s - could not find user profile for userId: %s", ExceptionType.RESOURCENOTFOUNDEXCEPTION, userId);
+            final String exceptionMsg = String.format("%s - could not find user profile", ExceptionType.RESOURCENOTFOUNDEXCEPTION);
```

Comment:
please make it void instead of boolean if you are not adding any logic based on return value.

Reply:
done